test/persistence-hooks.suite.js

```diff
@@ -1828,6 +2068,442 @@ module.exports = function(dataSource, should) {
       });
     });
 
+    describe('PersistedModel.replaceOrCreate', function() {
+      it('triggers hooks in the correct order on create', function(done) {
+        monitorHookExecution();
+
+        TestModel.replaceOrCreate(
+          { id: 'not-found', name: 'not found' },
+          function(err, record, created) {
+            if (err) return done(err);
+            triggered.should.eql([
+              'access',
+              'before save',
+              'persist',
+              'loaded',
+              'after save'
+            ]);
+            done();
+          });
+      });
+
+      it('triggers hooks in the correct order on replace', function(done) {
+        monitorHookExecution();
+
+        TestModel.replaceOrCreate(
+          { id: existingInstance.id, name: 'new name' },
+          function(err, record, created) {
+            if (err) return done(err);
+            if (dataSource.connector.replaceOrCreate) {
+              triggered.should.eql([
+                'access',
+                'before save',
+                'persist',
+                'loaded',
+                'after save'
+              ]); 
+            } else {
+              triggered.should.eql([
+                'access',
+                'loaded',
```

Comment:
Is this still true? As I commented above, since `findOne` is called with `options.notify:false`, I would expect it does not trigger the `loaded` hook?